save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/modal/ModalBuilder.kt

```diff
@@ -228,3 +257,98 @@ fun ChildrenBuilder.modalBuilder(
         }
     }
 }
+
+/**
+ * Universal function to create modals with click condition styles inside react modals
+ *
+ * @param onCloseButtonPressed callback that will be applied to `X` button in the top-right corner
+ * @param buttonBuilder lambda that generates several buttons, must contain either [button] or [buttonBuilder]
+ * @param clickBuilder lambda that generates several click in modal
+ * @param title of the modal that will be shown in top-left corner
+ * @param message main text that will be shown in the center of modal
+ * @param clickBuilder
+ */
+@Suppress("TOO_MANY_PARAMETERS", "LongParameterList")
+fun ChildrenBuilder.modalBuilderWithClick(
```

Comment:
Do you really need this method? I mean it seems you use this one only once as well as the one with three builders (the one below).

Reply:
I deleted 1 method
Now there are 2 of them left

Reply:
Now there is 1 added modal left